### DISCUSSION TITLE
AllenNLP v1.5.0

### DISCUSSION THREAD

**Discussion by vrunm** [explosion/spaCy#12226] [Help: Coding & Implementations]
I have been using AllenNLP v1.5.0 for migrating old code.
What is the version of spacy that works well AllenNLP v1.5.0 ?
I have tried using spacy 2.1.0 but still ran into some import issues.

**Comment by rmitsch** [comment]
Hi @vrunm, according to the [release notes](https://github.com/allenai/allennlp/releases/tag/v1.5.0) AllenNLP v1.5.0 seems to support spaCy 3.0. Have you tried that?

**Comment by vrunm** [comment]
Yes I have tried that it does not seem to work.

**Reply by vrunm** [reply]
Also I got an import error as :

    from allennlp.data.tokenizers.word_splitter import SpacyWordSplitter
    ModuleNotFoundError: No module named 'allennlp.data.tokenizers.word_splitter'

Also allennlp.data.tokenizers.word_splitter no longer exists in the latest version of AllenNLP.
It has been replaced with SpacyTokenizer. So what would be the correct version of Spacy to use for a tokenizer in AllenNLP v1.5.0?

**Reply by rmitsch** [reply]
`SpacyTokenizer` is part of the AllenNLP API, not spaCy's. This means that this is not related to spaCy directly, but (presumably) to breaking changes in between versions of AllenNLP. I advise you to consult the AllenNLP release/migration notes or ask on StackOverflow. Asking on the [AllenNLP discussion board](https://github.com/allenai/allennlp/discussions) is also an option - considering the project is read-only at this point, I wouldn't expect a response there though.

**Reply by vrunm** [reply]
Thanks @rmitsch will do that.
